plugin/core/sessions.py

```diff
@@ -193,6 +193,13 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
                             "source.organizeImports"
                         ]
                     }
+                },
+                "dataSupport": True,
+                "resolveSupport": {
+                    "properties": [
+                        "edit",
+                        "command"
+                    ]
```

Comment:
- What about `additionalTextEdits`?
- I don't think CompletionItems have `edit` property.

Reply:
I don't understand the question. This defines what properties of a code action structure we can resolve. We can resolve all properties (namely the `edit` and the `command`).

Reply:
I was thinking of and looking at the documentation of `completion/resolve` the whole time ;)